README.md

```diff
@@ -1,14 +1,114 @@
+Snapshot mode. Then, when all the records have been read, Source switches to CDC mode.
+
+### Snapshot
+
+The connector in the Snapshot mode reads all rows from the table in batches via SELECT with fetching and ordering.
+
+Example of a query:
+
+```
+SELECT {{columns...}}
+FROM {{table}}
+ORDER BY {{orderingColumn}}
+WHERE {{keyColumn}} > {{position.lastProcessdValue}}
+FETCH NEXT {{batchSize}} ROWS ONLY;
+```
+
+When all records have been returned, the connector switches to the CDC mode.
+
+### Change Data Capture
+
+This connector implements CDC features for Oracle by adding a tracking table and a trigger to populate it. The tracking
+table and trigger name have the same names as a target table wit the prefix `CONDUIT`. The tracking table has all the
+same columns as the target table plus three additional columns:
+
+| name                          | description                                          |
+|-------------------------------|------------------------------------------------------|
+| `CONDUIT_TRACKING_ID`         | Autoincrement index for the position.                |
+| `CONDUIT_OPERATION_TYPE`      | Operation type: `insert`, `update`, or `delete`.     |
+| `CONDUIT_TRACKING_CREATED_AT` | Date when the event was added to the tacking table.  |
+
+Every time data is added, changed, or deleted from the target table, this event will be written to the tracking table.
+
+The queries to get change data from the tracking table look pretty similar to queries in the Snapshot mode, but
+with `CONDUIT_TRACKING_ID` ordering column.
+
+The Ack method collects the `CONDUIT_TRACKING_ID` of those records that have been successfully applied, in order to
+remove them later in a batch (every 5 seconds or when the connector is closed).
+
+### Position
+
+Example of the position:
+
+```
+{
+  "mode": "snapshot",
+  "last_processed_val": 1
+}
+```
+
+`mode` is a mode of an iterator (`snapshot` and `cdc`);
+
+`last_processed_val` is a last processed element value of an ordering column.
+
+### Configuration Options
+
+| name             | description                                                                         | required |
+|------------------|-------------------------------------------------------------------------------------|----------|
+| `url`            | String line for connection to DB2.                                                  | **true** |
+| `table`          | The name of a table in the database that the connector should write to, by default. | **true** |
+| `keyColumn`      | Column name records should use for their `Key` fields.                              | **true** |
+| `orderingColumn` | Column name of a column that the connector will use for ordering rows.              | **true** |
+| `columns`        | List of column names that should be included in each Record's payload.              | false    |
+| `batchSize`      | Size of rows batch. Min is 1 and max is 100000.                                     | false    |
+
```

Comment:
please add block 
Known limitations:
Changing a table name during a connector update can cause quite unexpected results. Therefore, it's highly not recommended to do this.

Reply:
Deal.